### ISSUE TITLE
Urgent: Clarify that Game Boy is a Nintendo trademark

### ISSUE THREAD

**Issue by Exagone313** [taisel/IodineGBA#63]
See also https://github.com/github/dmca/blob/master/2018/2018-07-19-Nintendo.md

**Comment by Onimishra** [comment]
I’ve never heard about projects getting a DNC’s notice for referencing the Game Boy name, as long as it is not part of you own product name. 
Jsemu’s problem was that they had committed ROMs. These a protected and there are many cases of people being shut down because of infringement on this part. 

I think we are generally in the clear, as the linked dmca notice does not contain the words “game boy”. It wouldn’t hurt to put a “Game Boy is a trade mark of Nintendo Inc.” at the bottom of the .md, but I don’t think it’s urgent :)

**Comment by Exagone313** [comment]
Yes, this should be enough. But keep reading the news, they may come to the extinguish phase.

**Comment by Onimishra** [comment]
Could you provide sources for this? I’ve never heard about this before, and nintendo have been one of the companies that have given emulation some leeway throughout the years, as it keeps some of their old platforms alive and gives them free publicity.
